lm75bd/lm75bd.c

```diff
@@ -27,7 +27,31 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+
+  error_code_t errCode;
+  #define POINTER_VALUE 0 //pointer bit value B1 and B0
```

Comment:
move this to line 12 and call it LM75BD_REG_TEMP or something similar. Define constants usually appear near the start of the file and not in functions